assets/css/draw.css

```
@@ -0,0 +1,133 @@
+html {
+    margin: 0;
+    padding: 0;
+  }
+  
+  body {
+    height: 100vh;
+    width: 100vw;
+    overflow-y: hidden;
+    overflow-x: hidden;
+  }
+  
+  .color-wrapper {
+    position: absolute;
+    top: 80%;
+    left: 50%;
+    transform: translateX(-50%);
+    user-select: none;
+  }
+  
+  .size-wrapper {
+    position: absolute;
+    top: 87%;
+    left: 50%;
+    transform: translate(-50%);
+    user-select: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+  }
+  
+  .color {
+    width: 26px;
+    height: 26px;
+    border-radius: 50%;
+    cursor: pointer;
+    background-color: #fff;
+    display: inline-block;
+    margin: 10px;
+    transition: 250ms ease;
+  }
+  
+  .size {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 50px;
+    cursor: pointer;
+    margin: 10px;
+    margin-top: 5px;
+    transition: 250ms ease;
+  }
+  
+  .size button {
+    width: 70px;
+    padding: 10px 5px;
+    margin: 5px;
+    border-radius: 5px;
+    border: 1px solid black;
+  }
+  
+  .s-small:hover {
+    transform: scale(0.85);
```

I think it should be 
```suggestion
    transform: scale(0.8);
```